base/ca/shared/profiles/acmeServerCert.cfg

```diff
@@ -0,0 +1,121 @@
+policyset.serverCertSet.4.default.name=Authority Key Identifier Default
+policyset.serverCertSet.5.constraint.class_id=noConstraintImpl
+policyset.serverCertSet.5.constraint.name=No Constraint
+policyset.serverCertSet.5.default.class_id=authInfoAccessExtDefaultImpl
+policyset.serverCertSet.5.default.name=AIA Extension Default
+policyset.serverCertSet.5.default.params.authInfoAccessADEnable_0=true
+policyset.serverCertSet.5.default.params.authInfoAccessADLocationType_0=URIName
+policyset.serverCertSet.5.default.params.authInfoAccessADLocation_0=http://ocsp.example.com
+policyset.serverCertSet.5.default.params.authInfoAccessADMethod_0=1.3.6.1.5.5.7.48.1
+policyset.serverCertSet.5.default.params.authInfoAccessADEnable_1=true
+policyset.serverCertSet.5.default.params.authInfoAccessADLocationType_1=URIName
+policyset.serverCertSet.5.default.params.authInfoAccessADLocation_1=http://cert.example.com
+policyset.serverCertSet.5.default.params.authInfoAccessADMethod_1=1.3.6.1.5.5.7.48.2
+policyset.serverCertSet.5.default.params.authInfoAccessCritical=false
+policyset.serverCertSet.5.default.params.authInfoAccessNumADs=2
+policyset.serverCertSet.6.constraint.class_id=noConstraintImpl
+policyset.serverCertSet.6.constraint.name=No Constraint
+policyset.serverCertSet.6.default.class_id=userExtensionDefaultImpl
+policyset.serverCertSet.6.default.name=User supplied extension in CSR
+policyset.serverCertSet.6.default.params.userExtOID=2.5.29.17
+policyset.serverCertSet.7.constraint.class_id=validityConstraintImpl
+policyset.serverCertSet.7.constraint.name=Validity Constraint
+policyset.serverCertSet.7.constraint.params.range=90
+policyset.serverCertSet.7.constraint.params.notBeforeCheck=false
+policyset.serverCertSet.7.constraint.params.notAfterCheck=false
+policyset.serverCertSet.7.default.class_id=validityDefaultImpl
+policyset.serverCertSet.7.default.name=Validity Default
+policyset.serverCertSet.7.default.params.range=90
+policyset.serverCertSet.7.default.params.startTime=0
+policyset.serverCertSet.8.constraint.class_id=signingAlgConstraintImpl
+policyset.serverCertSet.8.constraint.name=No Constraint
+policyset.serverCertSet.8.constraint.params.signingAlgsAllowed=SHA256withRSA,SHA512withRSA,SHA256withEC,SHA384withRSA,SHA384withEC,SHA512withEC
+policyset.serverCertSet.8.default.class_id=signingAlgDefaultImpl
+policyset.serverCertSet.8.default.name=Signing Alg
+policyset.serverCertSet.8.default.params.signingAlg=-
+policyset.serverCertSet.9.constraint.class_id=noConstraintImpl
+policyset.serverCertSet.9.constraint.name=No Constraint
+policyset.serverCertSet.9.default.class_id=sanToCNDefaultImpl
+policyset.serverCertSet.9.default.name=SAN to CN Default
+policyset.serverCertSet.10.constraint.class_id=keyConstraintImpl
+policyset.serverCertSet.10.constraint.name=Key Constraint
+policyset.serverCertSet.10.constraint.params.keyType=RSA
+policyset.serverCertSet.10.constraint.params.keyParameters=1024,2048,3072,4096
```

Comment:
How about a minimum of 2048? :)

Reply:
Sure, let me update it.

BTW, this profile was based on an existing profile which has the same setting:
https://github.com/dogtagpki/pki/blob/master/base/ca/shared/profiles/ca/caServerCert.cfg#L33
We probably need to fix that in a separate PR.